doc/source/technology_showcase_examples/techdemo-25/ex_25-tecstent.py

```diff
@@ -0,0 +1,364 @@
+mapdl.keyopt(itype="3", knum="2", value="1")
+mapdl.keyopt(itype="3", knum="4", value="111111")
+mapdl.type(itype="3")
+
+mapdl.mat("2")
+mapdl.r(nset="4")
+mapdl.real(nset="4")
+mapdl.et(itype="5", ename="170")
+mapdl.et(itype="6", ename="174")
+mapdl.keyopt(itype="6", knum="12", value="5")
+mapdl.keyopt(itype="6", knum="4", value="1")
+mapdl.keyopt(itype="6", knum="2", value="2")
+mapdl.keyopt(itype="5", knum="2", value="1")
+mapdl.keyopt(itype="5", knum="4", value="111111")
+mapdl.type(itype="5")
+
+# %% [markdown]
+# #### Settings for standard contact between stent and inner plaque wall contact surface
+
+# %%
+mapdl.mp(lab="MU", mat="1", c0="0")
+mapdl.mat("1")
+mapdl.mp(lab="EMIS", mat="1", c0="7.88860905221e-31")
+mapdl.r(nset="6")
+mapdl.real(nset="6")
+mapdl.et(itype="10", ename="170")
+mapdl.et(itype="11", ename="177")
+mapdl.r(nset="6", r3="1.0", r4="1.0", r5="0")
+mapdl.rmore(r9="1.0E20", r10="0.0", r11="1.0")
+mapdl.rmore(r7="0.0", r8="0", r9="1.0", r10="0.05", r11="1.0", r12="0.5")
+mapdl.rmore(r7="0", r8="1.0", r9="1.0", r10="0.0")
+mapdl.keyopt(itype="11", knum="5", value="0")
+mapdl.keyopt(itype="11", knum="7", value="1")
+mapdl.keyopt(itype="11", knum="8", value="0")
+mapdl.keyopt(itype="11", knum="9", value="0")
+mapdl.keyopt(itype="11", knum="10", value="2")
+mapdl.keyopt(itype="11", knum="11", value="0")
+mapdl.keyopt(itype="11", knum="12", value="0")
+mapdl.keyopt(itype="11", knum="2", value="3")
+mapdl.keyopt(itype="10", knum="5", value="0")
+
+# %% [markdown]
+# #### Settings for MPC based, force-distributed constraint on proximal stent nodes
+
+# %%
+mapdl.mat("1")
+mapdl.r(nset="7")
+mapdl.real(nset="7")
+mapdl.et(itype="12", ename="170")
+mapdl.et(itype="13", ename="175")
+mapdl.keyopt(itype="13", knum="12", value="5")
+mapdl.keyopt(itype="13", knum="4", value="1")
+mapdl.keyopt(itype="13", knum="2", value="2")
+mapdl.keyopt(itype="12", knum="2", value="1")
+mapdl.keyopt(itype="12", knum="4", value="111111")
+mapdl.type(itype="12")
+
+# %% [markdown]
+# #### Settings for MPC based, force-distributed constraint on distal stent nodes
+
+# %%
+mapdl.mat("1")
+mapdl.r(nset="8")
+mapdl.real(nset="8")
+mapdl.et(itype="14", ename="170")
+mapdl.et(itype="15", ename="175")
+mapdl.keyopt(itype="15", knum="12", value="5")
+mapdl.keyopt(itype="15", knum="4", value="1")
+mapdl.keyopt(itype="15", knum="2", value="2")
+mapdl.keyopt(itype="14", knum="2", value="1")
+mapdl.keyopt(itype="14", knum="4", value="111111")
+mapdl.type(itype="14")
+
+# %% [markdown]
+# ### Reading the geometry file
+
+# %%
+mapdl.cdread(option="db", fname="stent", ext="cdb")
+mapdl.allsel(labt="all")
+mapdl.finish()
+
+# %% [markdown]
+# ## Static Analysis
+#
+# Run static analysis
+
+# %%
+# enter solution processor and define analysis settings
+mapdl.run("/solu")
+mapdl.antype(antype="0")
+mapdl.nlgeom(key="on")
+
+# %% [markdown]
+# ### Apply Load Step 1: Balloon angioplasty of the artery to expand it past the
+# radius of the stent - IGNORE STENT
```

Comment:
Are we keeping these??

Reply:
I can remove them if you prefer. I thought it was easier to have the comment - even for the Python file - but it's something that I can easily change.

Reply:
If the ``%% [markdown]`` is rendered by the html you should delete it. If it is not, then it is up to you. Other examples don't have them though.